server/index.js

```diff
@@ -155,6 +155,33 @@ app.post('/api/cart', (req, res, next) => {
     });
 });
 
+app.post('/api/orders', (req, res, next) => {
+  const cartId = req.session.cartId;
+  const name = req.body.name;
+  const creditCard = req.body.creditCard;
+  const shippingAddress = req.body.shippingAddress;
+
+  if (!cartId) {
+    next(new ClientError('Please create a new account.', 400));
+  } else if (!name || !creditCard || !shippingAddress) {
+    next(new ClientError('Please fill out the form completely.', 400));
+  }
```

Comment:
Don't forget to `return` if you're not good to go ahead with the database query.